wajibisha/templates/blocks/team.html

```diff
@@ -0,0 +1,42 @@
+{% load staticfiles %}
+<div class="showcase how-it-works text-center" style="height:;width:100%;">
+  <div class="wrapper clearfix text-center">
+   <h3 style="color:#0067A3;">THE TEAM</h3>
```

Comment:
If heading colors are consistent across the site, it should be changed in CSS not using `style`.